test/mp/test/foo.capnp

```diff
@@ -13,6 +13,7 @@ $Proxy.includeTypes("mp/test/foo-types.h");
 
 interface FooInterface $Proxy.wrap("mp::test::FooImplementation") {
     add @0 (a :Int32, b :Int32) -> (result :Int32);
+    addOut @19 (a :Int32, b :Int32, ret :Int32) -> (ret :Int32);
```

Comment:
In commit "test default PassField impl handles output parameters" (f3688fdaed1fddd52755857f2b068927f1a428a7)

Since the test is supposed to demonstrate using output parameters and is not using the input value of `ret` it would be good to make the declaration reflect that by simplifying:

```diff
-    addOut @19 (a :Int32, b :Int32, ret :Int32) -> (ret :Int32);
+    addOut @19 (a :Int32, b :Int32) -> (ret :Int32);
```

If you wanted to extend the test to cover input/output parameters, too, you could add another function like:

```c++
void addInOut(int x, int& sum) { sum += x; }
```

```capnp
addInOut @20 (x :Int32, sum :Int32) -> (sum :Int32);
```